parm/soca/soca_fix_stage_500.yaml.j2

```diff
@@ -9,8 +9,6 @@ copy:
 - ["{{ SOCA_INPUT_FIX_DIR }}/field_table", "{{ DATA }}/field_table"]
 - ["{{ SOCA_INPUT_FIX_DIR }}/diag_table", "{{ DATA }}/diag_table"]
 - ["{{ SOCA_INPUT_FIX_DIR }}/MOM_input", "{{ DATA }}/MOM_input"]
-- ["{{ SOCA_INPUT_FIX_DIR }}/fields_metadata.yaml", "{{ DATA }}/fields_metadata.yaml"]
```

Comment:
Is this functionality being replaced somewhere? Because the yet-unmerged LETKF uses it.

Reply:
@AndrewEichmann-NOAA , I missed it when reviewing then. You should not use ` fields_metadata.yaml` from the fix_dir. We need to have that file removed at some point.

Reply:
@AndrewEichmann-NOAA Like @guillaumevernieres said. The current ```fields_metadata.yaml``` is actually staged here https://github.com/NOAA-EMC/GDASApp/blob/34f894ffc8ff15b1a43d537a0a1043670231e5a3/parm/soca/soca_utils_stage.yaml.j2#L5